src/templates/open-enterprise/index.js

```diff
@@ -13,28 +13,40 @@ import {
 import header from './header.svg'
 import icon from './icon.svg'
 
+const ONE_PERCENT = new BN(10).pow(new BN(16))
+
 function completeDomain(domain) {
   return domain ? `${domain}.aragonid.eth` : ''
 }
 
 function adjustVotingSettings(support, quorum) {
   // The max value for both support and quorum is 100% - 1
-  const onePercent = new BN(10).pow(new BN(16))
-  const hundredPercent = onePercent.mul(new BN(100))
+  const hundredPercent = ONE_PERCENT.mul(new BN(100))
 
-  let adjustedSupport = onePercent.mul(new BN(support))
+  let adjustedSupport = ONE_PERCENT.mul(new BN(support))
   if (adjustedSupport.eq(hundredPercent)) {
     adjustedSupport = adjustedSupport.sub(new BN(1))
   }
 
-  let adjustedQuorum = onePercent.mul(new BN(quorum))
+  let adjustedQuorum = ONE_PERCENT.mul(new BN(quorum))
   if (adjustedQuorum.eq(hundredPercent)) {
     adjustedQuorum = adjustedQuorum.sub(new BN(1))
   }
 
   return [adjustedSupport.toString(), adjustedQuorum.toString()]
 }
 
+function adjustDotVotingSettings(dvSupport, dvQuorum) {
+  const adjustedDvSupport = ONE_PERCENT.mul(new BN(dvSupport))
+
+  // The min value for quorum must be 1 or greater
+  const adjustedDvQuorum = dvQuorum.isZero()
```

Comment:
I think this line is making it throw an error. After I click "Launch organization", I get:

```
t.isZero is not a function
in Unknown
in div
in t
in Onboarding___StyledDiv3
in div
in t
```